api/src/main/java/com/graphhopper/config/CHProfileConfig.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.graphhopper.config;
+
+import java.util.Objects;
+
+import static com.graphhopper.config.ProfileConfig.validateProfileName;
+
+public class CHProfileConfig {
+    private String profile = "unset_ch_profile";
```

Comment:
Its certainly not necessary, but I thought its a bit more explicit like that. I do not like using null, because then it could actually be null after deserialization (not sure if this is good or bad, but for example if we always make sure everything is initialized it would be sufficient to check for empty string (if even) and we do not need null checks. -> I changed it to be initialized with an empty string now.